From bf9a5b8e5be78bfac6954850fee1c156eca0f1b7 Mon Sep 17 00:00:00 2001
From: Radostin Stoyanov <rstoyanov@fedoraproject.org>
Date: Mon, 10 Jan 2022 10:37:52 +0000
Subject: [PATCH 107/120] contributing: remove old badges and logo

CI badges and logo are already present in the readme file.

Signed-off-by: Radostin Stoyanov <rstoyanov@fedoraproject.org>
---
 CONTRIBUTING.md | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
index 96972296e..864caf93e 100644
--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -1,8 +1,3 @@
-[![master](https://travis-ci.org/checkpoint-restore/criu.svg?branch=master)](https://travis-ci.org/checkpoint-restore/criu)
-[![development](https://travis-ci.org/checkpoint-restore/criu.svg?branch=criu-dev)](https://travis-ci.org/checkpoint-restore/criu)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/55251ec7db28421da4481fc7c1cb0cee)](https://www.codacy.com/app/xemul/criu?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=xemul/criu&amp;utm_campaign=Badge_Grade)
-<p align="center"><img src="https://criu.org/w/images/1/1c/CRIU.svg" width="128px"/></p>
-
 ## How to contribute to CRIU
 
 CRIU project is (almost) the never-ending story, because we have to always keep up with the
-- 
2.34.1

